script.js

```diff
@@ -1,74 +1,93 @@
 document.addEventListener("DOMContentLoaded", () => {
-  const getData = async () => {
-    const response = await fetch("https://jsonplaceholder.typicode.com/users");
-    const users = await response.json();
-    return users;
+  const getData = async (url) => {
+    try {
+      const response = await fetch(url);
+      const users = await response.json();
+      return users;
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  const createEl = (el, text, styles) => {
```

Comment:
Pass arguments like an object. It'll help you to get rid of passing `null`, etc.